File: java/java-common/src/main/java/com/showmeyourcode/kafka/java/common/KafkaProperties.java
package com.showmeyourcode.kafka.java.common;

public final class KafkaProperties {
    public static final String TOPIC = "java-example-topic";
    // If you register a new Schema, you need to use a different topic.
    public static final String TOPIC2 = "java-example-topic2";
    public static final String BOOTSTRAP_SERVERS = "localhost:9092";

    public static final String CONSUMER_GROUP_ID = "java-test-group";
    public static final String PRODUCER_CLIENT_ID = "JavaKafkaProducer";
    public static final String PRODUCER_AVRO_CLIENT_ID = "JavaKafkaAvroProducer";

    public static final String AVRO_SCHEMA_REGISTRY = "http://localhost:8081";

    private KafkaProperties() {
    }
}

File: common/src/main/kotlin/com/showmeyourcode/kafka/common/KafkaProperties.kt
package com.showmeyourcode.kafka.common

import org.apache.kafka.clients.producer.ProducerConfig
import org.apache.kafka.common.serialization.LongSerializer
import org.apache.kafka.common.serialization.StringSerializer
import java.util.*

class KafkaProperties {
    companion object {
        const val TOPIC = "my-example-topic"

        /*
        If you have more brokers, use comma as a separator for instance:
        const val BOOTSTRAP_SERVERS = "localhost:9092,localhost:9093"
         */
        const val BOOTSTRAP_SERVERS = "localhost:9092"
    }
}
